test/test_utils/ecs.py

```diff
@@ -809,6 +823,10 @@ def setup_ecs_inference_service(
             "deviceName": "device_1",
             "deviceType": ei_accelerator_type
         }
+
+    if processor == "neuron":
```

Comment:
lets also check if `num_neurons` is not empty

Reply:
Done